src/DataStructureGroupParser.h

```diff
@@ -0,0 +1,154 @@
+//
+//  DataStructureGroupParser.h
```

Comment:
would be good you you could `git mv` the files next time when you are renaming, so I can actually see what has been changed.